src/services/api/utils.ts

```diff
@@ -12,6 +12,7 @@ function parseMessageTpl(messageTpl: string) {
   }
 }
 
+// TODO: fix issue where this needs sign from the wallet
```

Comment:
no prio todo?

Reply:
I fix this todo in [e393c56](https://github.com/dappforce/grillchat/pull/434/commits/e393c56e36e5ebfb44ba28b52998b2c45b40ce2e)